tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -77,6 +77,13 @@ std::string ROOT::Experimental::RNTupleInspector::GetName()
    return descriptorGuard->GetName();
 }
 
+std::uint64_t ROOT::Experimental::RNTupleInspector::GetNEntries()
+{
+   fPageSource->Attach();
```

Comment:
This should be done conditionally, i.e. if it wasn't attached already, no?  I guess we don't make any guarantees on repeated calls to `RPageSource::Attach()`.